armi/settings/fwSettings/globalSettings.py

```diff
@@ -33,87 +33,88 @@
 CONF_DEBUG = "debug"
 CONF_DEBUG_MEM = "debugMem"
 CONF_DEBUG_MEM_SIZE = "debugMemSize"
+CONF_DECAY_CONSTANTS = "decayConstants"
 CONF_DEFAULT_SNAPSHOTS = "defaultSnapshots"
+CONF_DEFERRED_INTERFACE_NAMES = "deferredInterfaceNames"
+CONF_DEFERRED_INTERFACES_CYCLE = "deferredInterfacesCycle"
 CONF_DETAIL_ALL_ASSEMS = "detailAllAssems"
 CONF_DETAIL_ASSEM_LOCATIONS_BOL = "detailAssemLocationsBOL"
 CONF_DETAIL_ASSEM_NUMS = "detailAssemNums"
-CONF_DUMP_SNAPSHOT = "dumpSnapshot"
-CONF_PHYSICS_FILES = "savePhysicsFiles"
+CONF_DETAILED_AXIAL_EXPANSION = "detailedAxialExpansion"
 CONF_DO_ORIFICED_TH = "doOrificedTH"  # zones
+CONF_DUMP_SNAPSHOT = "dumpSnapshot"
 CONF_EQ_DIRECT = "eqDirect"  # fuelCycle/equilibrium coupling
+CONF_EXPLICIT_REPEAT_SHUFFLES = "explicitRepeatShuffles"
+CONF_FLUX_RECON = "fluxRecon"  # strange coupling in fuel handlers
 CONF_FRESH_FEED_TYPE = "freshFeedType"
 CONF_GEOM_FILE = "geomFile"
-CONF_START_CYCLE = "startCycle"
-CONF_LOADING_FILE = "loadingFile"
-CONF_START_NODE = "startNode"
+CONF_INDEPENDENT_VARIABLES = "independentVariables"
+CONF_INITIALIZE_BURN_CHAIN = "initializeBurnChain"
+CONF_INPUT_HEIGHTS_HOT = "inputHeightsConsideredHot"
 CONF_LOAD_STYLE = "loadStyle"
+CONF_LOADING_FILE = "loadingFile"
 CONF_LOW_POWER_REGION_FRACTION = "lowPowerRegionFraction"  # reports
+CONF_MATERIAL_NAMESPACE_ORDER = "materialNamespaceOrder"
+CONF_MIN_MESH_SIZE_RATIO = "minMeshSizeRatio"
 CONF_MODULE_VERBOSITY = "moduleVerbosity"
 CONF_MPI_TASKS_PER_NODE = "mpiTasksPerNode"
 CONF_N_CYCLES = "nCycles"
-CONF_TIGHT_COUPLING = "tightCoupling"
-CONF_TIGHT_COUPLING_MAX_ITERS = "tightCouplingMaxNumIters"
-CONF_TIGHT_COUPLING_SETTINGS = "tightCouplingSettings"
-CONF_CYCLES_SKIP_TIGHT_COUPLING_INTERACTION = "cyclesSkipTightCouplingInteraction"
+CONF_NON_UNIFORM_ASSEM_FLAGS = "nonUniformAssemFlags"
+CONF_NUM_PROCESSORS = "numProcessors"
 CONF_OPERATOR_LOCATION = "operatorLocation"
+CONF_OUTPUT_CACHE_LOCATION = "outputCacheLocation"
 CONF_OUTPUT_FILE_EXTENSION = "outputFileExtension"
+CONF_PHYSICS_FILES = "savePhysicsFiles"
 CONF_PLOTS = "plots"
 CONF_POWER = "power"
+CONF_POWER_FRACTIONS = "powerFractions"
+CONF_PROFILE = "profile"
+CONF_REALLY_SMALL_RUN = "reallySmallRun"
 CONF_RUN_TYPE = "runType"
-CONF_EXPLICIT_REPEAT_SHUFFLES = "explicitRepeatShuffles"
 CONF_SKIP_CYCLES = "skipCycles"
 CONF_SMALL_RUN = "smallRun"
-CONF_REALLY_SMALL_RUN = "reallySmallRun"
+CONF_START_CYCLE = "startCycle"
+CONF_START_NODE = "startNode"
 CONF_STATIONARY_BLOCK_FLAGS = "stationaryBlockFlags"
+CONF_T_IN = "Tin"
+CONF_T_OUT = "Tout"
 CONF_TARGET_K = "targetK"  # lots of things use this
+CONF_TIGHT_COUPLING = "tightCoupling"
+CONF_TIGHT_COUPLING_MAX_ITERS = "tightCouplingMaxNumIters"
+CONF_TIGHT_COUPLING_SETTINGS = "tightCouplingSettings"
+CONF_TRACE = "trace"
 CONF_TRACK_ASSEMS = "trackAssems"
+CONF_UNIFORM_MESH_MINIMUM_SIZE = "uniformMeshMinimumSize"
+CONF_USER_PLUGINS = "userPlugins"
 CONF_VERBOSITY = "verbosity"
+CONF_VERSIONS = "versions"
```

Comment:
Love the alphabetizing here.

Reply:
Yeah, I couldn't help myself.  I didn't _have_ to do the cleanup. But there were ~50 constants all in no perceptible order.  So hard to read and understand.